examples/notebooks/introduction_enlyze_python_sdk.ipynb

```diff
@@ -0,0 +1,2981 @@
+       "      <td>2022-05-28 18:59:40+02:00</td>\n",
+       "      <td>338.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>99703</th>\n",
+       "      <td>2022-05-28 16:59:50+00:00</td>\n",
+       "      <td>2022-05-28 18:59:50+02:00</td>\n",
+       "      <td>338.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>99704 rows × 3 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                           time                time_local  \\\n",
+       "0     2022-05-17 04:00:00+00:00 2022-05-17 06:00:00+02:00   \n",
+       "1     2022-05-17 04:00:10+00:00 2022-05-17 06:00:10+02:00   \n",
+       "2     2022-05-17 04:00:20+00:00 2022-05-17 06:00:20+02:00   \n",
+       "3     2022-05-17 04:00:30+00:00 2022-05-17 06:00:30+02:00   \n",
+       "4     2022-05-17 04:00:40+00:00 2022-05-17 06:00:40+02:00   \n",
+       "...                         ...                       ...   \n",
+       "99699 2022-05-28 16:59:10+00:00 2022-05-28 18:59:10+02:00   \n",
+       "99700 2022-05-28 16:59:20+00:00 2022-05-28 18:59:20+02:00   \n",
+       "99701 2022-05-28 16:59:30+00:00 2022-05-28 18:59:30+02:00   \n",
+       "99702 2022-05-28 16:59:40+00:00 2022-05-28 18:59:40+02:00   \n",
+       "99703 2022-05-28 16:59:50+00:00 2022-05-28 18:59:50+02:00   \n",
+       "\n",
+       "       Extruder A Schmelzedruck  \n",
+       "0                         387.0  \n",
+       "1                         388.0  \n",
+       "2                         388.0  \n",
+       "3                         388.0  \n",
+       "4                         388.0  \n",
+       "...                         ...  \n",
+       "99699                     338.0  \n",
+       "99700                     338.0  \n",
+       "99701                     338.0  \n",
+       "99702                     338.0  \n",
+       "99703                     338.0  \n",
+       "\n",
+       "[99704 rows x 3 columns]"
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data['time_local'] = data['time'].dt.tz_convert('Europe/Berlin')\n",
+    "data[['time','time_local', 'Extruder A Schmelzedruck']]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0239a449-efc0-4cae-a233-5987c6a45968",
+   "metadata": {},
+   "source": [
+    "### For large periods and more fine-grained control - get_timeseries_with_resampling()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88bd9db6-6561-480f-bc96-ccddbe68cbcd",
+   "metadata": {},
+   "source": [
+    "The `get_timeseries` method returns the time series in the currently highest resolution (10 seconds). For a long period and many variables, this quickly results in a large amount of data. Due to the size of the data, there might be a noticeable delay in receiving the results, and in some cases, it may even cause a request timeout.\n",
```

Comment:
```suggestion
    "The `get_timeseries` method returns the time series data in the currently highest resolution (10 seconds). For a long period and many variables, this quickly results in a large amount of data. Due to the size of the data, there might be a noticeable delay in receiving the results, and in some cases, it may even cause a request timeout.\n",
```

Reply:
2d5a669